packages/server/src/api/controllers/static/index.ts

```diff
@@ -24,14 +24,69 @@ import AWS from "aws-sdk"
 import fs from "fs"
 import sdk from "../../../sdk"
 import * as pro from "@budibase/pro"
-import { App, Ctx, ProcessAttachmentResponse } from "@budibase/types"
+import { UserCtx, App, Ctx, ProcessAttachmentResponse } from "@budibase/types"
 import {
   getAppMigrationVersion,
   getLatestMigrationId,
 } from "../../../appMigrations"
 
 import send from "koa-send"
 
+const getThemeVariables = theme => {
```

Comment:
spectrum css isn't loaded for the skeleton rendered in the initial client doc yet, so we need to insert the necessary styles here.